components/wpt-results.html

```diff
@@ -180,23 +180,6 @@
         this._refreshDisplayedSpecs()
       }
 
-      _specFilter(spec) {
-        if (this.query.length === 0) {
-          return true
-        }
-
-        const matchesQuery = (tf) => tf.testFile.toLowerCase().includes(this.query.toLowerCase())
-        return spec.testFiles.some(matchesQuery)
-      }
-
-      _testFileFilter(testFile) {
-        if (this.query.length < 3) {
-          return false
-        }
```

Comment:
This was the thing that was making it so that individual files aren't displayed until the query is at length 3, right? What's the behavior after this CL?